assets/scss/drupal/module/_views.scss

```diff
@@ -0,0 +1,28 @@
+.view {
+  td .dropbutton-single .dropbutton-widget {
+    // Inherit instead of absolute to avoid the button to be partially hidden.
+    position: inherit;
+  }
+
+  // Reduce chance of secondary actions being truncated when dropbutton is in last table column.
+  .dropbutton .dropbutton-action.secondary-action > * {
+    // Allow line breaks in secondary dropbutton actions.
+    white-space: initial;
+    font-size: 90%;
+  }
+
+  .view-filters .form-actions {
+    // For .form-actions:last-child it is set to 0. But For the "Apply" button
+    // in View filters to be on the same height as the input fields it has to be
+    // 30px here, too.
+    margin-bottom: 30px;
+  }
+}
+
+// Don't truncate secondary actions when dropbutton is in last table row.
+#page-content {
+  overflow: initial !important;
+}
```

Comment:
This is being defined in `_article.scss` of the base theme, which gets included before the sub theme stylesheets, so this should not need the `!important`.